### ISSUE TITLE
Server crashing on latest journeymap

### ISSUE THREAD

**Issue by dmodoomsirius** [TeamJM/journeymap#19]
#### JourneyMap Version:

`Example: 1.12.2-5.6.0b2`

#### Forge Version:

`Example: 1.12.214.23.1.2589`

#### Describe the problem in repeatable steps:

1. load up server get crash. https://gist.github.com/dmodoomsirius/6708540f399cc303626bb226305a39bc

#### Relevant errors from `.minecraft/journeymap/journeymap.log`:
did not generate.

**Comment by Renari** [comment]
Strange, I'm running the same version of Journeymap on my server and am not having this issue.

My Forge Version: `14.23.1.25.85`

journeymap.server.player.global.config:
> // JourneyMap server configuration file. Modify at your own risk!
> // To restore the default settings, simply delete this file before starting Minecraft server
> // For more information, go to: http://journeymap.info/JourneyMapServer
> //
> // Global Server Player Configuration : Applies to all dimensions unless overridden. 
> {
>   "useWorldId": "false",
>   "actionTeleport": "false",
>   "displayCompass": "true",
>   "displayFullscreen": "true",
>   "displayMinimap": "true",
>   "displayWaypointBeacon": "true",
>   "displayWaypointManager": "true",
>   "displayWebmap": "true",
>   "maptypeBiome": "true",
>   "maptypeDay": "true",
>   "maptypeNight": "false",
>   "maptypeTopo": "false",
>   "maptypeUnderground": "false",
>   "radarHostileMob": "false",
>   "radarNPC": "false",
>   "radarPassiveMob": "false",
>   "radarPlayer": "true",
>   "radarVehicle": "true",
>   "radarWaypoint": "true",
>   "configVersion": "5.6.0b1"
> }

journeymap.server.op.global.config:

> // JourneyMap server configuration file. Modify at your own risk!
> // To restore the default settings, simply delete this file before starting Minecraft server
> // For more information, go to: http://journeymap.info/JourneyMapServer
> //
> // Global Server Op Configuration : Applies to all dimensions unless overridden. 
> {
>   "useWorldId": "false",
>   "actionTeleport": "false",
>   "displayCompass": "true",
>   "displayFullscreen": "true",
>   "displayMinimap": "true",
>   "displayWaypointBeacon": "true",
>   "displayWaypointManager": "true",
>   "displayWebmap": "true",
>   "maptypeBiome": "true",
>   "maptypeDay": "true",
>   "maptypeNight": "false",
>   "maptypeTopo": "false",
>   "maptypeUnderground": "false",
>   "radarHostileMob": "false",
>   "radarNPC": "false",
>   "radarPassiveMob": "false",
>   "radarPlayer": "true",
>   "radarVehicle": "true",
>   "radarWaypoint": "true",
>   "configVersion": "5.6.0b1"
> }
>

**Comment by Misunderstood-Wookiee** [comment]
Same problem here - crash on startup if the server configs are generated, if I delete the server config folder and start the server it will start normally however after a clean shutdown and consecutive re-start produces crash before world load. 

https://pastebin.com/VUmVzYDt

**Comment by BrainStone** [comment]
Duplicate of #18
